aodh/server.sls

```diff
@@ -1,4 +1,22 @@
 {%- from "aodh/map.jinja" import server with context %}
-{%- if server.enabled %}
+{%- if server.enabled and server.version == "mitaka" %}
```

Comment:
Indeed. I was wondering about 'server.version == "mitaka"'. Adding this test will force you to update it for every new OpenStack release.